sympy/stats/tests/test_continuous_rv.py

```diff
@@ -848,6 +848,31 @@ def test_maxwell():
     assert cdf(X)(x) == erf(sqrt(2)*x/(2*a)) - sqrt(2)*x*exp(-x**2/(2*a**2))/(sqrt(pi)*a)
     assert diff(cdf(X)(x), x) == density(X)(x)
 
+def test_Moyal():
+    mu = Symbol('mu',real=False)
+    sigma = Symbol('sigma', real=True, positive=True)
+    raises(ValueError, lambda: Moyal('M',mu, sigma))
+
+    mu = Symbol('mu', real=True)
+    sigma = Symbol('sigma', real=True, negative=True)
+    raises(ValueError, lambda: Moyal('M',mu, sigma))
+
+    sigma = Symbol('sigma', real=True, positive=True)
+    M = Moyal('M', mu, sigma)
+    assert density(M)(z) == sqrt(2)*exp(-exp((mu - z)/sigma)/2
+                        - (-mu + z)/(2*sigma))/(2*sqrt(pi)*sigma)
+    assert cdf(M)(z).simplify() == 1 - erf(sqrt(2)*exp((mu - z)/(2*sigma))/2)
+    assert characteristic_function(M)(z) == 2**(-I*sigma*z)*exp(I*mu*z) \
+                        *gamma(-I*sigma*z + Rational(1, 2))/sqrt(pi)
+    assert E(M) == mu + EulerGamma*sigma + sigma*log(2)
+    assert moment_generating_function(M)(z) == 2**(-sigma*z)*exp(mu*z) \
+                        *gamma(-sigma*z + Rational(1, 2))/sqrt(pi)
+
+    M = Moyal('M', 1, 2)
+    assert E(M) == 1 + 2*EulerGamma + 2*log(2)
+    assert variance(M).simplify() == 2*pi**2
+    assert kurtosis(M).simplify() == 7
+    assert skewness(M).simplify() == 28*sqrt(2)*zeta(3)/pi**3
```

Comment:
IMO, these can be removed as symbolic tests are already there.

Reply:
Are they affecting coverage in any way?

Reply:
No, they are not affecting the coverage.